btr-api/src/btr_api/services/entity.py

```diff
@@ -111,6 +110,43 @@ def get_entity_info(self, user_jwt: JwtManager, path: str, token: str = None) ->
             self.app.logger.debug('Legal-api integration failure: %s', repr(err))
             raise ExternalServiceException(error=repr(err), status_code=HTTPStatus.INTERNAL_SERVER_ERROR) from err
 
+    def get_entity_todos(self, business_identifier: str, user_jwt: JwtManager):
```

Comment:
Hey hrvoje I don't think we want this to have its own method since we already have 'get_entity_info' which is meant for this sort of thing (just need to pass that one the path for todos)